spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -0,0 +1,229 @@
+        await createDatabaseAsyncSample.CreateDatabaseAsyncAsync(ProjectId, InstanceId, DatabaseId);
+        await insertDataAsyncSample.InsertDataAsync(ProjectId, InstanceId, DatabaseId);
+        await addColumnAsyncSample.AddColumnAsync(ProjectId, InstanceId, DatabaseId);
+        await addIndexAsyncSample.AddIndexAsync(ProjectId, InstanceId, DatabaseId);
+        // Add storing Index on table.
+        await addStoringIndexAsyncSample.AddStoringIndexAsync(ProjectId, InstanceId, DatabaseId);
+        // Update the value of MarketingBudgets.
+        await RefillMarketingBudgetsAsync(300000, 300000);
+    }
+
+    private async Task InitializeBackupAsync()
+    {
+        // Sample database for backup and restore tests.
+        try
+        {
+            CreateDatabaseAsyncSample createDatabaseAsyncSample = new CreateDatabaseAsyncSample();
+            InsertDataAsyncSample insertDataAsyncSample = new InsertDataAsyncSample();
+            await createDatabaseAsyncSample.CreateDatabaseAsyncAsync(ProjectId, InstanceId, BackupDatabaseId);
+            await insertDataAsyncSample.InsertDataAsync(ProjectId, InstanceId, BackupDatabaseId);
+        }
+        catch (Exception e) when (e.ToString().Contains("Database already exists"))
+        {
+            // We intentionally keep an existing database around to reduce
+            // the likelihood of test timeouts when creating a backup so
+            // it's ok to get an AlreadyExists error.
+            Console.WriteLine($"Database {BackupDatabaseId} already exists.");
+        }
+
+        try
+        {
+            CreateBackupSample createBackupSample = new CreateBackupSample();
+            createBackupSample.CreateBackup(ProjectId, InstanceId, BackupDatabaseId, BackupId);
+        }
+        catch (RpcException e) when (e.StatusCode == StatusCode.AlreadyExists)
+        {
+            // We intentionally keep an existing backup around to reduce
+            // the likelihood of test timeouts when creating a backup so
+            // it's ok to get an AlreadyExists error.
+            Console.WriteLine($"Backup {BackupId} already exists.");
+        }
+        catch (RpcException e) when (e.StatusCode == StatusCode.FailedPrecondition
+        && e.Message.Contains("maximum number of pending backups (1) for the database has been reached"))
+        {
+            // It's ok backup has been in progress in another test cycle.
+            Console.WriteLine($"Backup {BackupId} already in progress.");
+        }
+    }
+
+    private async Task DeleteDatabaseAsync(string databaseId)
+    {
+        string adminConnectionString = $"Data Source=projects/{ProjectId}/instances/{InstanceId}";
+        using (var connection = new SpannerConnection(adminConnectionString))
+        {
+            using (var cmd = connection.CreateDdlCommand($@"DROP DATABASE {databaseId}"))
+            {
+                await cmd.ExecuteNonQueryAsync();
+            }
+        }
```

Comment:
nit: since C#8 you can do as follows. The using scope becomes the same as the declaration scope. Maybe consider doing this in the samples as well.
```suggestion
        using var connection = new SpannerConnection(adminConnectionString)
        using var cmd = connection.CreateDdlCommand($@"DROP DATABASE {databaseId}")
        await cmd.ExecuteNonQueryAsync();
```